src/lib.rs

```diff
@@ -75,6 +83,39 @@ pub(crate) fn build_vector<Vector: VelloVector + Component>() -> SystemConfigs {
         .into_configs()
 }
 
+pub(crate) fn build_head<HeadEquipt: VelloVector + VectorBorder + Component>() -> SystemConfigs {
+    append_heads::<HeadEquipt>.into_configs()
+}
+
+#[allow(clippy::type_complexity)]
+fn append_heads<HeadEquipt: VelloVector + VectorBorder + Component>(
+    mut q_vectors: Query<
+        (&HeadEquipt, &Head, &mut VelloScene),
+        (Without<Stroke>, Or<(Changed<HeadEquipt>, Changed<Fill>)>),
+    >,
+    shapes: Res<Shapes>,
+    time: Res<Time>,
+) {
+    let time = time.elapsed_seconds();
```

Comment:
The `time` should be a user defined value in the `Head` component. It represents the percentage we want  the head to be at between the start and end of the border of a particular shape.

Reply:
Let's say we have a line:
```
---------------
```
When `time` = 0.0:
```
>--------------
```
When `time` = 1.0:
```
-------------->
```
When `time` = 0.5:
```
------->-------
```

Reply:
Oh sorry, it seems Ive misinterpreted the `VectorBorder` idea. will be fixed in the next commit!